Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/BuildEvidenceParameter.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2009-2016, United States Government, as represented by the Secretary of Health and Human Services.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *     * Redistributions of source code must retain the above
+ *       copyright notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the documentation
+ *       and/or other materials provided with the distribution.
+ *     * Neither the name of the United States Government nor the
+ *       names of its contributors may be used to endorse or promote products
+ *       derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE UNITED STATES GOVERNMENT BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package gov.hhs.fha.nhinc.callback.openSAML;
+
+import java.util.List;
+import org.joda.time.DateTime;
+import org.opensaml.saml.saml2.core.AttributeStatement;
+import org.opensaml.saml.saml2.core.Subject;
+
+public class BuildEvidenceParameter {
+
+    private CallbackProperties properties;
+    private String evAssertionID;
+    private DateTime issueInstant = null;
+    private String format = null;
+    private DateTime beginValidTime = null;
+    private DateTime endValidTime = null;
+    private String issuer = null;
+    private List<AttributeStatement> statements;
+    private Subject subject;
+
+    public BuildEvidenceParameter(String evAssertionID, String issuer, CallbackProperties properties,
+        List<AttributeStatement> statements, Subject subject) {
+        this.evAssertionID = evAssertionID;
+        this.properties = properties;
+        this.statements = statements;
+        this.subject = subject;
+        this.issuer = issuer;
+        setProperties(properties);
+    }
+
+    public CallbackProperties getProperties() {
+        return properties;
+    }
+
+    public void setProperties(CallbackProperties properties) {
```

Comment:
why do we need to create these new variables?  Why can't you just have one getter for properties and I can get these variables from there, such as buildEvidenceParameter.getCalBackProperties().getIssueInstant()?  Seems very redundant.